babel.config.js

```diff
@@ -3,59 +3,44 @@ module.exports = {
     [
       '@babel/preset-env',
       {
-        useBuiltIns: 'usage',
-        corejs: 3,
-        targets: { node: '8' },
+        targets: { node: 18 },
       },
     ],
     '@babel/preset-typescript',
   ],
   plugins: [
-    '@babel/plugin-proposal-optional-chaining',
-    '@babel/plugin-proposal-nullish-coalescing-operator',
-    '@babel/plugin-proposal-class-properties',
```

Comment:
these 3 plugins are deprecated (all 3 proposals reached stage 4 and are in the ECMA spec).
if you do `npm show <pkg-name>` you'll see there's babel transform plugins to handle the new syntax, all of those are already included in `@babel/preset-env` so we don't need to manually add them as devDep.